src/components/MainView/main-view.jsx

```diff
@@ -1,10 +1,19 @@
-import { useState, useEffect } from "react"; 
+import { useState, useEffect } from "react";
 import { MovieCard } from "../MovieCard/movie-card.jsx";
 import { MovieView } from "../MovieView/movie-view.jsx";
+import { LoginView } from "../login-view/login-view";
+import { SignupView } from "../signup-view/signup-view";
+import { NavigationBar } from "../navigation-bar/navigation-bar";
+import { ProfileView } from "../profile-view/profile-view";
+import { Row, Col } from "react-bootstrap";
```

Comment:
This line is causing the app to crash. You trying to import components from a package that is not installed or listed as a dependency in `package.json`. Please add the package.